parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -181,11 +176,7 @@ installStandaloneContainerd() {
 
     #if there is no containerd_version input from RP, use hardcoded version
     if [ -z "${CONTAINERD_VERSION}" ]; then
-        # pin 18.04 to 1.7.1
         CONTAINERD_VERSION="1.7.15"
```

Comment:
huh, I thought we use components.json for this now, didn't realize we were still hard-coding containerd version here...

Reply:
right it might be dead code? @Devinwong  are you aware why we still have this her e?